lib/perl/Genome/Model/Build/Command/Export.pm

```diff
@@ -30,6 +37,18 @@ sub execute {
     $self->check_available_space($allocation);
 
     my $export_directory = $self->export_directory;
+    my $tempdir;
+    if($self->create_tarball){
+        #create temp directory for pre-tarball files
+        $tempdir = Genome::Sys->create_temp_directory();
+        unless($tempdir) {
+            $self->error_message("Unable to create temporary file $!");
+            die;
```

Comment:
* This would be better as `die $self->error_message` or even better as `$self->fatal_message`.
* Note that `$!` is examined in `Genome::Sys`, so this is unlikely to ever be reached:
https://github.com/genome/genome/blob/cd160fd03c34189ea7a362a8afc8e395a9843fc4/lib/perl/Genome/Sys.pm#L941-L954